### DISCUSSION TITLE
WinForms Charting: LiveCharts deprecated, considering OxyPlot — Any Alternatives?

### DISCUSSION THREAD

**Discussion by mar1111284** [community/community#70102] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Currently working on a WinForms student project (an expense manager) with .NET 6.0, I need to find an alternative to LiveCharts as it's deprecated.

After some research, i'm using OxyPlot, but it is not very customizable, i'm looking for something that match with modern GUI

my current graphics tab :

![qxqxq](https://github.com/community/community/assets/76847464/6f15e5ed-3045-4fc6-9791-46ca600bd304)

THX!

code link: https://github.com/marinVcq/wallet-manager-Csharp

**Comment by ebubekirgungor** [comment]
Hello, you can use LiveCharts 2.0 but it is currently in release candidate. Here is [winforms documentation](https://livecharts.dev/docs/WinForms/2.0.0-rc1/Overview.Installation)

**Reply by mar1111284** [reply]
Okay, thank you. I will take a look, and in the worst case, I will use user controls.
